source/mainnet/net/voting/gc-voting.rst

```diff
@@ -0,0 +1,74 @@
+
+.. image:: ../images/voting/pre-election.png
+    :alt: diagram showing steps below
+    :width: 50%
+
+#. The Election coordinator uses the coordinator tool to get the initial weight values and initialize an instance of the election smart contract.
+
+#. Each guardian sends their account address to Concordium.
+
+#. Candidates are nominated and can "campaign". Candidates **must** submit a JSON file with information such as name, an image or logo, and the URL of a site with information about their campaign. This information is shown in the voting dApp.
+
+#. The Election coordinator adds the guardian addresses to the smart contract along with election parameters such as the start and end date and time, and candidates, and deploys and initializes the smart contract.
+
+#. The election server is started, the indexer is started, and the Guardian app is built and released.
+
+#. Guardians download the guardian app and install it. They export their account keys from their wallet and import that file to the Guardian app to generate their key pair. And they generate encrypted shares for each guardian. If any invalid submissions from peers are detected, the guardian files a complaint which requires manual intervention from the election coordinator by setting up a new election.
+
+During the election
+===================
+
+The period of time during the election is the :term:`election phase`.
+
+.. image:: ../images/voting/election.png
+    :alt: diagram showing steps below
+    :width: 50%
+
+#. Voters open the dApp and cast their votes. The voter must connect their wallet and sign and submit the transaction to register it on the blockchain. In the case of delegated votes, it doesn't matter when voting power is delegated, as long as it happens within the voting "window", i.e., between election start and end time. The latest delegation registration counts and you cannot redelegate voting power delegated to you; only your initial weight can be delegated.
+
+#. The votes are registered in the smart contract and the server reads the votes from the contract.
+
+After the election
+==================
+
+After the election is the :term:`tally phase`.
+
+.. image:: ../images/voting/post-election.png
+    :alt: diagram showing steps below
+    :width: 50%
+
+#. Once the election closes, the election coordinator uses the coordinator tool to get the final weights (the initial weights plus weights after taking vote delegation into account) and compute the encrypted tally which is registered in the smart contract.
+
+#. The guardians use the Guardian app to generate their decryption share of the tally and post that before the deadline specified. They then generate and register proof of correct decryption. If proof generation fails, manual intervention is required by the election coordinator to reset the tally phase with the option of excluding any number of guardians.
```

Comment:
If we exclude too many guardians, decryption is not possible anymore. So we can't exclude any number.
```suggestion
#. The guardians use the Guardian app to generate their decryption share of the tally and post that before the deadline specified. They then generate and register proof of correct decryption. If proof generation fails, manual intervention is required by the election coordinator to reset the tally phase with the option of excluding faulty guardians.
```